usaspending_api/api_contracts/contracts/subawards/Subawards.md

```diff
@@ -0,0 +1,56 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Subawards
+
+These endpoints are used to power USAspending.gov's subaward listings.
+
+# Subaward List
+
+This endpoint returns a filtered set of subawards.
+
+## Subawards [/api/v2/subawards/]
+
+This endpoint returns a list of data that is associated with the award profile page.
+
+### Subawards [POST]
+
++ Request (application/json)
+    + Attributes (object)
+        + page (required, number)
+            + Default: 1
+        + limit (optional, number)
+            + Default: 10
+        + filters (required, FilterObject)
+        + order (required, enum[string], fixed-type)
+            + Members
+                + asc
+                + desc 
+            + Default: desc
+            
++ Response 200 (application/json)
+    + Attributes
+        + results(required, array[SubawardResponse], fixed-type)
+        + `page_metadata`(required, PageMetadataObject)
+
+# Data Structures
+
+## SubawardResponse (object)
++ `subaward_number` (required, string)
++ amount (required, number)
++ id (required, number)
++ `action_date` (required, string)
++ `recipient_name` (required, string)
++ description (required, string)
+
+## PageMetadataObject (object)
++ page (required, number)
++ next (required, number, nullable)
++ previous (required, number, nullable)
++ hasNext (required, boolean)
++ hasPrevious (required, boolean)
+
+## FilterObject (object)
```

Comment:
This Filter Object doesn't look correct for this endpoint. According to the code it matches the shared pagination fields used by a number of Advanced Search endpoints, (limit, order, page, sort) and not be a nested object.

Reply:
Experimentally, it seems to allow both filters but I'm unsure if the v1 filter does anything. This endpoint may have developed issues while out of use. I'll update the contracts to just use the v2.